tests/update_eve_image/testdata/update_eve_image.txt

```diff
@@ -35,13 +37,18 @@ eden -t 10m controller edge-node eveimage-update file://{{EdenConfigPath "eve.di
 # Check stderr, it must be empty
 ! stderr .
 
-# Run monitoring of Info messages to obtain info with PartitionState inprogress or active and previously defined ShortVersion
+# Run monitoring of Info messages to obtain info with PartitionState active and previously defined ShortVersion
 message 'Waiting for EVE update...'
-{{$test}} -out InfoContent.dinfo.SwList[0].ShortVersion 'InfoContent.dinfo.SwList[0].PartitionState:inprogress|active InfoContent.dinfo.SwList[0].ShortVersion:{{ $short_version }}'
+{{$test}} -out InfoContent.dinfo.SwList[0].ShortVersion 'InfoContent.dinfo.SwList[0].PartitionState:active InfoContent.dinfo.SwList[0].ShortVersion:{{ $short_version }}'
 
 # Check stdout of previous command. Expected to get previously defined ShortVersion
 stdout '{{ $short_version }}'
 
+# Reset EVE version
+test eden.escript.test -test.run TestEdenScripts/revert_eve_image_update -test.v -testdata {{EdenConfig "eden.tests"}}/update_eve_image/testdata/
```

Comment:
This file includes another one?

Reply:
Here I call another escripts, which reset EVE to the version defined in the global config.